src/cascadia/TerminalSettingsModel/FontConfig.h

```diff
@@ -23,6 +23,9 @@ Author(s):
 #include "IInheritable.h"
 #include <DefaultSettings.h>
 
+using IFontAxesMap = winrt::Windows::Foundation::Collections::IMap<winrt::hstring, int32_t>;

```

Comment:
so axes are signed whole numbers, not floats? but we use floats later in DxFontRenderData..?

Reply:
or, are we saying that users should not need to specify floating point axes, because all the axes they can access are integers?

Reply:
You're right, these should be floats. Fixed